### DISCUSSION TITLE
量化交易之数字货币历史数据收集

### DISCUSSION THREAD

**Discussion by giscus** [zizhuspot/caoyuan#5] [General]
# 量化交易之数字货币历史数据收集

对于低频或者中频交易来说，收集历史行情数据是开始量化交易的第一步，只有拥有足够的历史数据，你才能方便的回测自己的代码。也才有足够的信心上实盘

https://www.quant1098.com/1f67a9b980a1/

**Comment by youngjunning** [comment]
专业
